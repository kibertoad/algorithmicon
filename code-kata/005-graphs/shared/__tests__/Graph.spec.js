describe('Graph', () => {
  it('Depth-first traversal', () => {})

  it('Breadth-first traversal', () => {})
})
