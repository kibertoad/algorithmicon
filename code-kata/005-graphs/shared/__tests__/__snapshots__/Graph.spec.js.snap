// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graph Creates vertices and edges correctly 1`] = `
"a -> b 
b -> a c 
c -> b 
"
`;
